### ISSUE TITLE
Command to quit the reflection

### ISSUE THREAD

**Issue by meedstrom** [emacsomancer/org-roam-daily-reflection#1]
I'm thinking a quit could 

1. restore the state of the windows from before, and 
2. attempt to kill the buffers that were opened.

It's not that I'm in the habit of "cleaning" my buffer lists, I spent some years with `C-x k` unbound.  It's just that I'd like to hook something like `(lambda () (text-scale-decrease 1))` for this particular purpose, and then I want to kill those buffers when I'm done because it'd be weird later on to run across these buffers with the small text.

What do you think?

**Comment by emacsomancer** [comment]
I've been planning (1), and it should be pretty easy to implement as I've already got a window config restore operation that's used for the splitting function itself (in case it can't actually manage to split the frame properly). I hadn't thought about (2), but it's a good idea, and probably wouldn't be very hard either: make a list of buffers that were opened in the process of reflecting (maybe check to see that they weren't open before?) and kill them if they weren't modified.

**Comment by emacsomancer** [comment]
does  https://github.com/emacsomancer/org-daily-reflection/commit/1a028f08f96201f7ca963e8279c6660ebbd94012  resolve this?

**Comment by meedstrom** [comment]
Good idea! The toggle works, but not the close-reflection-newly-opened. Does it for you?

**Comment by meedstrom** [comment]
Value of `org-daily-reflection--list-of-newly-opened-entries` is non-nil, it has 6 buffers. Stays that way after evalling `(org-daily-reflection-close-reflection-newly-opened)`.

edit: looks like it's effectively a no-op ;) PR opened.

**Comment by emacsomancer** [comment]
> Value of `org-daily-reflection--list-of-newly-opened-entries` is non-nil, it has 6 buffers. Stays that way after evalling `(org-daily-reflection-close-reflection-newly-opened)`.
> 
> edit: looks like it's effectively a no-op ;) PR opened.

Thanks! 

Merged PR. Does it seem to work for you now?

**Comment by emacsomancer** [comment]
In https://github.com/emacsomancer/org-daily-reflection/commit/42a8da2a20c4ac31248072ce17803c0a579d11f6 , fixed up `org-daily-reflection-close-reflection-newly-opened`, which can be used interactively to close any buffer that the package opened that haven't been modified (also see customisation `org-daily-reflection-close-unmodified-newly-opened-buffers` which will call this automatically when the command `org-daily-reflection-restore-prior-windows` (to restore the prior layout) is called.

Closing this issue for now as it seems to function as expected, but feel free to re-open if there are remaining issues.
